src/modules/dashboard/components/Dialogs/GnosisControlSafeDialog/GnosisControlSafeDialog.tsx

```diff
@@ -114,7 +130,15 @@ const validationSchema = (isPreview) =>
         data: yup.string().when('transactionType', {
           is: (transactionType) =>
             transactionType === TransactionTypes.RAW_TRANSACTION,
-          then: yup.string().required(() => MSG.requiredFieldError),
+          then: yup
+            .string()
+            .required(() => MSG.requiredFieldError)
+            .matches(/^0x/, () => MSG.startWith0x)
```

Comment:
How come you need to specifically check that the string starts with 0x? Isn't `.isHexString` enough?

Reply:
@ArmandoGraterol Yes, can do. Was trying to be as explicit as possible but I'll follow your lead in the abi parser branch.